common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -129,6 +129,18 @@
     "message" : [
       "<value1> <symbol> <value2> caused overflow."
     ],
+    "subClass" : {
+      "WITHOUT_SUGGESTION" : {
```

Comment:
Are there any tests for this sub-class? If not, please, add it.

Reply:
For now this path cannot happen, as all occurrences hit the WITH_SUGGESITON, I can totally remove the subclasses and just leave it with the WITH_SUGGESTION path.

Reply:
sgtm. In that case, you can fold the sub-class error message w/ parent's one, and eliminate the sub-class at all.